doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -7,7 +7,26 @@ Operating with a resource constrained host
    :local:
    :depth: 2
 
-This guide provides recommendations and guidelines for using the nRF7002 as a companion chip on resource-constrained hosts such as the nRF5340 SoC.
+This guide provides recommendations and guidelines for using the nRF70 series as a companion chip on resource-constrained hosts such as the nRF5340 SoC.
+
+Networking Stack combinations
+*****************************
+The table below indicates the support of various classes of Wi-Fi networking applications (denoted by the corresponding protocol stack capabilities) running on the different nRF host MCUs combined with the nRF70 Series companion IC. The table should help users identify and select the appropriate host MCU depending on the IoT use-case
```

Comment:
Sorry @krish2718 my bad - the text I proposed earlier was, obviously, for the second table introduced here.

My proposal for this intro text, here is as follows:

"The table below lists the required protocol stack combinations for various  classes of Wi-Fi internetworking applications."